mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -56,3 +69,33 @@ impl BridgeController {
         future::pending::<()>().await;
     }
 }
+
+impl Default for BridgeController {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct BridgeControllerHandle {
+    sender: UnboundedSender<BridgeControllerUpdate>,
+}
+
+impl BridgeControllerHandle {
+    pub fn send(&mut self, message: BridgeControllerUpdate) -> Result<(), Error> {
+        self.sender
+            .send(message)
+            .map_err(Error::SendControllerMessage)
+    }
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct BridgeControllerUpdate {
+    // TODO: add settings
```

Comment:
are you going to address this in the next PR?

Reply:
yes